### ISSUE TITLE
Change PRCDA and UIHO to use CountyAtDxAnalysis

### ISSUE THREAD

**Issue by howew** [imsweb/algorithms#118]
These calculations use reported county at DX.  CDC has asked that they be changed to CountyAtDxAnalysis.

**Comment by depryf** [comment]
I can take a shot at this at some point. Or if you want to expediate it, you can also submit a pull request. Either way.

**Comment by howew** [comment]
No rush on this.  I just told them it would be done sometime before the fall submissions.

**Comment by depryf** [comment]
@howew  I am making this change now.

Do I need to change the version of the algorithms?

Here is what it says now:

```java
    public static final String ALG_NAME = "NPCR PRCDA & UIHO Linkage Program";
    public static final String ALG_VERSION = "2.0";
    public static final String ALG_INFO = "NPCR PRCDA & UIHO Linkage Program version 2.0, released in July 2020";
```

**Comment by howew** [comment]
Yes.  Version 2.1 seems appropriate.  Thanks.

**Comment by depryf** [comment]
This is done. Will be available with version 3.4
